archimedes/src/app/component/image/image.component.html

```diff
@@ -0,0 +1,79 @@
+<div *ngFor="let image of images" class="col-sm-6 col-md-4 col-xl-3">
+
+    <a [attr.href]="['#modal-' + image.id]" data-toggle="modal">
+        <img src="{{image.thumbnail_url}}" alt="{{image.location}}" class="img-fluid mb-3">
+    </a>
+
+    <div class="modal" id="modal-{{image.id}}" tabindex="-1" role="dialog"
+        [attr.aria-labelledby]="['modal-' + image.id]" aria-hidden="true">
```

Comment:
aria-labelledby needs to point to some other element that describes the modal's contents--might be better to use aria-label here instead, e.g. `[attr.aria-label]="Image details for [image.filename]"`